res/lang/news/news_en.txt

```diff
@@ -20,95 +20,95 @@ NEWSSTUDIO_SUBSCRIPTION_LEVEL                      = subscription level
 TOASTMESSAGE_BOMB_DETONATION_IN_TVTOWER_TEXT = A bomb detonation damaged the room %ROOM%.
 TOASTMESSAGE_BOMB_RENOVATION_COST_TEXT       = Renovation will cost %COST%.
 TOASTMESSAGE_BOMB_LICENCE_DESTROYED_TEXT     = The explosion destroyed '%TITLE%'.
 TOASTMESSAGE_BOMB_CONCEPT_DESTROYED_TEXT     = The explosion destroyed a production plan.
-BOMB_DETONATION_IN_TVTOWER_NO_CLUES          = Attack on TVTower: No clues found
-BOMB_DETONATION_IN_TVTOWER_NO_CLUES_TEXT     = Public authorities investigated after the attack in the famous TVTower. Unfortunately they weren't able to find any useful clues.
+BOMB_DETONATION_IN_TVTOWER_NO_CLUES          = Attack on TVTower: No traces
+BOMB_DETONATION_IN_TVTOWER_NO_CLUES_TEXT     = After the attack in the well-known TV Tower, the authorities investigated. Unfortunately, no usable evidence could be secured.
 BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES       = Attack on TVTower: Clues found
-BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = Public authorities investigated after the attack in the famous TVTower - with success! Apparently resident TV channels (%CHANNELS%) assisted unwillingly with their roomsign-switch-games.
+BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = After the attack at the well-known TVTower, the authorities investigated - with success! Apparently, resident broadcasters (%CHANNELS%) contributed to the disaster with their sign-swapping games.
 
 
-NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_TITLE       = New movie by and with %DIRECTORNAME1%
-NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_DESCRIPTION = Director and main actor %DIRECTORNAME1% has new movie «%MOVIETITLE%» in the line.
+NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_TITLE       = New film made by and starring %DIRECTORNAME1%
+NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_DESCRIPTION = Director and main actor %DIRECTORNAME1% brings movie «%MOVIETITLE%» to the screen.
 
 NEWS_ANNOUNCE_MOVIE_NO_CAST_TITLE       = Soon in the movies: «%MOVIETITLE%»
 NEWS_ANNOUNCE_MOVIE_NO_CAST_DESCRIPTION = First trailer for «%MOVIETITLE%» published.
 
 NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_TITLE       = Soon in the movies: «%MOVIETITLE%»
-NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_DESCRIPTION = Main actor %ACTORNAME1% soon in the movies with «%MOVIETITLE%».
+NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_DESCRIPTION = Main actor %ACTORNAME1% soon to be featured in theaters with «%MOVIETITLE%».
 
 NEWS_ANNOUNCE_MOVIE_NO_ACTOR_TITLE       = Soon in the movies: «%MOVIETITLE%»
-NEWS_ANNOUNCE_MOVIE_NO_ACTOR_DESCRIPTION = Director %DIRECTORNAME1% lands «%MOVIETITLE%» to a cinema near you.
+NEWS_ANNOUNCE_MOVIE_NO_ACTOR_DESCRIPTION = Director %DIRECTORNAME1% brings film «%MOVIETITLE%» to a cinema near you.
 
 NEWS_ANNOUNCE_MOVIE_TITLE1 = New movie with %ACTORNAME1%
-NEWS_ANNOUNCE_MOVIE_TITLE2 = %DIRECTORNAME1% directs new movie
-#NEWS_ANNOUNCE_MOVIE_TITLE3 = 
+NEWS_ANNOUNCE_MOVIE_TITLE2 = %DIRECTORNAME1% shoots new film
+#NEWS_ANNOUNCE_MOVIE_TITLE3 = %DIRECTORNAME1% with new film in theaters
```

Comment:
If you add a previously missing translation, remove the `#` at the beginning of the line (otherwise commented out)

Reply:
I knew that, but too easy to miss.